common/utils/localStorage.js

```diff
@@ -18,7 +18,7 @@ export const saveState = (state: Object) => {
     const serializedState = JSON.stringify(state);
     localStorage.setItem(REDUX_STATE, serializedState);
   } catch (err) {
-    // TODO - log warning? or bubble?
+    console.warn(' Warning: corrupted local storage');
```

Comment:
leading space OCDs me :D, also i think this message is factually incorrect, we can use something like 'failed to save to local storage' or something. Load case is about getting something broken from storage, but here we either got object that isnt serializable or browser failed to save to localstorage for w/e reason